includes/class-wc-taxjar-integration.php

```diff
@@ -689,7 +695,7 @@ public function calculate_tax( $options = array() ) {
 			// Add shipping tax rate
 			$taxes['rate_ids']['shipping'] = $this->create_or_update_tax_rate(
 				$location,
-				$taxes['tax_rate'] * 100,
+				$taxes['shipping_rate'] * 100,
 				'',
 				$taxes['freight_taxable']
```

Comment:
Missing trailing comma, for consistency w/ other lines in file

Reply:
The correct way is actually not to have a trailing comma. All these formatting/styling inconsistencies will be fixed in a future PR. Due to the number of them I want to do a standalone PR for them so that any functional changes do not get lost in the PR.